cmd/antrea-agent/options.go

```diff
@@ -225,6 +232,26 @@ func (o *Options) validateFlowExporterConfig() error {
 			}
 			o.pollInterval = flowPollInterval
 		}
+		// Parse the given activeFlowExportTimeout config
+		if o.config.ActiveFlowExportTimeout != "" {
+			o.activeFlowTimeout, err = time.ParseDuration(o.config.ActiveFlowExportTimeout)
+			if err != nil {
+				return fmt.Errorf("ActiveFlowExportTimeout is not provided in right format")
+			}
+			if o.activeFlowTimeout < o.pollInterval {
+				return fmt.Errorf("ActiveFlowExportTimeout should be greater than or equal to FlowPollInterval")
+			}
```

Comment:
do you think this is worth returning an error for? maybe we should just set `o.activeFlowTimeout` to `o.pollInterval` and log a warning.

Reply:
I wanted to do the same thing but forgot to do that. Thanks for pointing; it definitely makes sense to make `activeFlowTimeout` to `pollInterval`. Fixed and repeated for `idleFlowTimeout` as well.